### ISSUE TITLE
Windfury Harpies in Mulgore not using spells

### ISSUE THREAD

**Issue by sh1tdev** [StygianTheBest/StygianCore#69]
I imagine this must be an AzerothCore issue but the Windfury Wind Witch, Windfury Matriarch and Windfury Sorceress are not using their spells in combat. I checked the DB and only the Windfury Sorceress was missing her spells (which should be Frostbolt spell id:116/205 - the two entries both appear the same but probably different in some way) both Wind Witch and Matriarch are simply not using theirs.

**Comment by StygianTheBest** [comment]
Yes, this is a database issue that exists in the core. There are hundreds if not thousands of these little issues, and you can see a large list of them over at the AzerothCore github. When I update the core I try to pull in all any fixes that have been applied.

Crosspost this issue to the AzerothCore github if you can so they can assist.
https://github.com/azerothcore/azerothcore-wotlk/issues
